src/components/Profile/EditProfileModal.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  IconButton,
+  Typography,
+  Box,
+  TextField,
+  DialogActions,
+  Button,
+} from "@mui/material/";
+import CloseIcon from "@mui/icons-material/Close";
+import { useState } from "react";
+import { DatePicker, LocalizationProvider } from "@mui/x-date-pickers/";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import axios from "axios";
+import { useAppSelector } from "../../state/hooks";
+import { EditableProfileContents } from "../../pages/Profile";
+
+const styles = {
+  dialog: {
+    borderRadius: 5,
+  },
+  dialogContent: {
+    display: "flex",
+    flexDirection: "column",
+    padding: 0,
+  },
+  dialogTitle: {
+    paddingBottom: 0,
+    paddingX: 0.5,
+    paddingTop: 1,
```

Comment:
Let's make the padding for Y: 1 so that it is even
```suggestion
    paddingX: 0.5,
    paddingY: 1,
```